easypost/__init__.py

```diff
@@ -77,7 +77,8 @@ def convert_to_easypost_object(response, api_key, parent=None, name=None):
         'PickupRate': PickupRate,
         'PostageLabel': PostageLabel,
         'CarrierAccount': CarrierAccount,
-        'User': User
+        'User': User,
+        'Report': Report,
```

Comment:
These lists are still incomplete: you need to add "ShipmentReport" as class type here and "shprep" to the prefixes list below so that they are properly interpreted by the client as Report objects. Right now it is just parsing Reports retrieved from the system as `easypost.EasyPostObject` not `easypost.Report`.

Same for the other report types, of course